app/models.py

```diff
@@ -1837,6 +1837,8 @@ def serialize(self):
 MANAGE_API_KEYS = 'manage_api_keys'
 PLATFORM_ADMIN = 'platform_admin'
 VIEW_ACTIVITY = 'view_activity'
+CREATE_BROADCASTS = 'create_broadcasts'
+APPROVE_BROADCASTS = 'approve_broadcasts'
```

Comment:
I'm not looking properly at this PR but just worth pointing out that I assume this means that the approve broadcasts permission also is permission to reject broadcasts too?

Wonder if it is worth making that explicit in the name of it?

Note, I don't feel too strongly about this. @quis might have opinions

Reply:
Yes, it would let you approve, cancel or reject. These names will be hard to change once added, so it would be good to get them right now - I can't think of a good name which covers all three actions.

Reply:
Interesting. I hadn't assumed it would also give them permission to cancel an alert...

Reply:
I think creators or approvers should be able to cancel. Is it worth making it a separate permission in the DB if it’s going to be ambiguous rolling it into another permission?

Reply:
Now updated as discussed on Slack to add separate `cancel_broadcasts` and `reject_broadcasts` permissions.